modules/UI/recording/Recording.js

```diff
@@ -451,7 +421,12 @@ var Recording = {
         }
         case Status.AVAILABLE:
         case Status.OFF: {
-            if (this.recordingType === 'jibri')
+            if (this.recordingType === 'jibri_file') {
+                this.eventEmitter.emit(
+                    UIEvents.RECORDING_TOGGLED);
```

Comment:
I honestly feel like UIEvents.RECORDING_TOGGLED should be killed off with this commit. It's only used to talk back to conference.js at this point. Maybe it can be a function/action...not sure on implementation but it seems killable now.